pymove/visualization/folium.py

```diff
@@ -0,0 +1,2042 @@
+        slice_event_show=slice_event_show,
+        slice_subject_show=slice_subject_show
+    )
+
+
+def create_geojson_features_line(
+    df_, label_lat='lat',
+    label_lon='lon', label_datetime='datetime'
+):
+    """
+    Create geojson features.
+
+    Parameters:
+    -----------
+    df_: DataFrame.
+        Input trajectory data.
+    label_datetime: string, optional, default 'datetime'.
+        date_time colunm label.
+    label_lat: string, optional, default 'lat'.
+        latitude column label.
+    label_lon: string, optional, default 'long'.
+        longitude column label.
+    """
+    print('> Creating GeoJSON features...')
+    features = []
+
+    row_iterator = df_.iterrows()
+    _, last = next(row_iterator)
+    colums = df_.columns
+
+    for i, row in tqdm(row_iterator, total=df_.shape[0] - 1) :
+        last_time = last[label_datetime].strftime('%Y-%m-%dT%H:%M:%S')
+        next_time = row[label_datetime].strftime('%Y-%m-%dT%H:%M:%S')
+
+        popup_list = [i + ': ' + str(last[i]) for i in colums]
+        popup1 = '<br>'.join(popup_list)
+        # 'id: '+last['id']+'<br>timestamp: '+last_time+'<br>lat:
+        # '+str(row['lat'])+'<br>lon: '+str(row['lon'])
+        # +'<br>activity: '+ row['y_unique']
+        # popup2 = 'id: '+row['id']+'<br>timestamp: '+next_time+'<br>lat:
+        # '+str(row['lat'])+'<br>lon: '+str(row['lon'])
+
+        feature = {
+            'type': 'Feature',
+            'geometry': {
+                'type': 'LineString',
+                'coordinates': [
+                    [last['lon'], last['lat']],
+                    [row['lon'], row['lat']]
+                ]
+
+            },
+            'properties': {
+                'times': [last_time, next_time],
+                'popup': popup1,
+                'style': {
+                    'color': 'red',
+                    'icon': 'circle',
+                    'iconstyle': {
+                        'color': 'red',
+                        'weight': 4
+                        # 'radius': row['Numerical_data']*40
+                    }
+                }
+            }
+        }
+        _, last = i, row
+
+        features.append(feature)
+
+    return features
+
+
+def plot_traj_timestamp_geo_json(
+    df_, label_datetime='datetime',
+    label_lat='lat', label_lon='lon',
+    tiles='cartodbpositron'
+):
+    """
+    Plot trajectories wit geo_json.
+
+    Parameters:
+    -----------
+    df_: DataFrame.
+        Input trajectory data.
+    label_datetime: string, optional, default 'datetime'.
+        date_time colunm label.
+    label_lat: string, optional, default 'lat'.
+        latitude column label.
+    label_lon: string, optional, default 'long'.
+        longitude column label.
+    tiles: string, optional, default 'cartodbpositron'.
+        folium tiles.
+    """
+    features = create_geojson_features_line(df_, label_datetime)
+    print('creating folium map')
+    map_ = create_base_map(
```

Comment:
Codacy found an issue: [No value for argument 'move_data' in function call](https://app.codacy.com/gh/InsightLab/PyMove/pullRequest?prid=5714604)